Modules/Registration/Common/include/itkLandmarkBasedTransformInitializer.h

```diff
@@ -192,6 +196,9 @@ class ITK_TEMPLATE_EXPORT LandmarkBasedTransformInitializer : public Object
   /** Initializer for VersorRigid3D */
   void
   InternalInitializeTransform(VersorRigid3DTransformType *);
+  /** Initializer for Similarity3DTransform */
```

Comment:
Although there are many other docstrings that do not end with a period (p. 373, 374), at least for the ones that are added/reworked, it'd be nice to add it following the ITK SW Guide.